src/test/java/test/integration/support/assertions/ArquillianConfigAssert.java

```diff
@@ -64,5 +83,14 @@ private String getContentOfNode(String property, List<Node> nodes) {
                 .findFirst()
                 .orElseThrow(() -> new IllegalStateException("Node with property: " + property + " not found."));
         }
+
+        public ObjectArrayAssert<String> withProperty(String key) {
+            final String contentOfNode = getContentOfNode(key, actual.getChildren());
+            Assertions.assertThat(contentOfNode).isNotEmpty();
+
+            final String[] strings = contentOfNode.trim().split("\n");
```

Comment:
We create configuration file with new lines using `System.lineSeparator()`. I'm afraid splitting by `\n` would not always work.